ui/selectors/confirm-transaction.js

```diff
@@ -218,7 +206,7 @@ export const sendTokenTokenAmountAndToAddressSelector = createSelector(
 );
 
 export const contractExchangeRateSelector = createSelector(
-  contractExchangeRatesSelector,
+  (state) => getTokenExchangeRates(state),
   tokenAddressSelector,
   (contractExchangeRates, tokenAddress) => {
     return contractExchangeRates[
```

Comment:
Code below could probably look up via the checksummed address instead of the O(n) case insensitive search